app/models/cas_seeds/match_decision_reasons.rb

```diff
@@ -159,5 +163,17 @@ def run!
         ::MatchDecisionReasons::MitigationDecline.where(name: reason_name).first_or_create!
       end
     end
+
+    private def create_case_contact_assigns_manager_decline_reasons!
```

Comment:
Did you mean to call these in `run!` or holding of until there is something in the constants?

Reply:
guess I missed that.